### PR TITLE
Fix css of Sitemap(#209)

### LINKED ISSUE #209: Sitemap is unstyled
It doesn't need a major overhaul, but OOTB the title is positioned to close to the header and too small.

### PR DESCRIPTION
**Pull request by iRohitSingh**
Fix #209 

Screenshot 2023-11-09 at 4 57 49 PM

### CODE REVIEW

**Comment by iRohitSingh** [comment]
@sneridagh I have appllied the css for sitemap but it is not center to the container.


Screenshot 2023-11-09 at 5 09 00 PM

**Review by sneridagh — CHANGES_REQUESTED** [review]
@iRohitSingh probably the container it's using is the wrong (Volto's) one. Take a look at

https://github.com/plone/volto/blob/ad9e98633c68d810b0201ba4e9e1c9c8b83e16cf/src/components/theme/View/LinkView.jsx#L25

**Comment by iRohitSingh** [comment]
> @iRohitSingh probably the container it's using is the wrong (Volto's) one. Take a look at
> 
> https://github.com/plone/volto/blob/ad9e98633c68d810b0201ba4e9e1c9c8b83e16cf/src/components/theme/View/LinkView.jsx#L25

@sneridagh regarding this comment even our new a container with default container width applied look like this which is not looking good. can we ask our designer for mockup and then implement it?

Screenshot 2023-11-17 at 5 24 41 PM

File: src/theme/main.scss [hunk 1]
```diff
@@ -31,5 +31,6 @@
 @import 'temp';
 
 @import 'addonsThemeCustomizationsMain';
+@import 'sitemap';
```
Comment (ichim-david) [inline src/theme/main.scss:34]: @iRohitSingh @sneridagh  I thought tat addonsThemeCustomizationMain should be the last import in order to get modification from other addons, thus the sitemap import shouldn't be placed here

Comment (sneridagh) [reply src/theme/main.scss:34]: @ichim-david it is... good catch. @iRohitSingh there can't be anything from that onwards.

Comment (sneridagh) [reply src/theme/main.scss]: @ichim-david fixed. @iRohitSingh please take alook that all is good now.

File: src/theme/_sitemap.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.section-sitemap {
+  #page-sitemap {
+    h1 {
+      font-size: $heading1;
+      font-weight: $medium;
+      line-height: 77px;
```
Comment (sneridagh) [inline src/theme/_sitemap.scss:6]: Why are we not using the corresponding mixin from the theme for h1?

**Review by sneridagh — CHANGES_REQUESTED** [review]

File: src/theme/_sitemap.scss (hunk 2 above)
Comment (sneridagh) [reply src/theme/_sitemap.scss:6]: why this need to be redefined?

Comment (iRohitSingh) [reply src/theme/_sitemap.scss]: Fixed

**Pull request merged**